cicd/pipelines/using-pods-in-a-privileged-security-context.adoc

```diff
@@ -24,7 +24,9 @@ In addition, the `Buildah` cluster task, shipped as part of the OpenShift Pipeli
 
 include::modules/op-running-pipeline-and-task-run-pods-with-privileged-security-context.adoc[leveloffset=+1]
 
+include::modules/op-running-pipeline-run-and-task-run-with-custom-scc-and-service-account.adoc[leveloffset=+1]
 
+[id=additional-references_{context}]
```

Comment:
please follow the guidelines for headings for additional resources: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies

I also noticed the main title of this doc uses:
`[id='using-pods-in-a-privileged-security-context']`

it should be like
`[id="using-pods-in-a-privileged-security-context"]`

Please double check any of the headings on these topics and use the guidelines going forward.